src/Services/SwaggerService.php

```diff
@@ -79,7 +80,9 @@ public function __construct(Container $container)
             $this->data = $this->driver->getProcessTmpData();
 
             if (empty($this->data)) {
-                $this->data = $this->generateEmptyData();
+                $this->data = empty($this->config['info'])
+                    ? $this->generateEmptyData()
+                    : $this->generateEmptyData($this->config['info']['description']);
```

Comment:
please change the `generateEmptyData` logic, this construction should be kept in the same state 

```suggestion
                $this->data = $this->generateEmptyData();
```